helix-core/src/test/java/org/apache/helix/model/TestResourceConfig.java

```diff
@@ -34,6 +34,11 @@
   private static final ObjectMapper _objectMapper = new ObjectMapper();
 
   @Test
+  public void testHead() {
```

Comment:
Again, curious about this trick. Wait for the explanation.

Reply:
This has been a while. I don't recall specific details. 

It seems to be that the test case may be spread over other test classes, which is hard to track, say if thread leak or something.

Reply:
assign order